.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 sudo: false
 node_js:
-  - "7"
-  - "6"
-  - "4"
+  - "16"
```

Comment:
Is Travis even online any more? If not, this could just be deleted. If it is stil online, maybe 18-24 here (per currently supported Node versions).